thirdeye-ui/src/app/rest/dto/workspace.interfaces.ts

```diff
@@ -15,3 +15,18 @@
 export interface Workspace {
     id: string | null;
 }
+
+export interface WorkspaceConfiguration {
```

Comment:
does this mean this will have to be updated anytime the schema is updated in the backend ?

Reply:
yes for consistency purpose in the code, but it won't break anything. Since all these are build time checks only

Reply:
ok thanks